NCS.DSS.EmploymentProgression/GetEmploymentProgressionById/Function/EmploymentProgressionGetByIdTrigger.cs

```diff
@@ -0,0 +1,110 @@
+            IJsonHelper jsonHelper,
+            IResourceHelper resourceHelper,
+            ILoggerHelper loggerHelper
+            )
+        {
+            _httpResponseMessageHelper = httpResponseMessageHelper;
+            _httpRequestHelper = httpRequestHelper;
+            _employmentProgressionGetByIdTriggerService = EmploymentProgressionGetByIdTriggerService;
+            _jsonHelper = jsonHelper;
+            _resourceHelper = resourceHelper;
+            _loggerHelper = loggerHelper;
+        }
+
+        [FunctionName(FunctionName)]
+        [Response(HttpStatusCode = (int)HttpStatusCode.OK, Description = "Employment progression found.", ShowSchema = true)]
+        [Response(HttpStatusCode = (int)HttpStatusCode.NoContent, Description = "Customer resource does not exist", ShowSchema = false)]
+        [Response(HttpStatusCode = (int)HttpStatusCode.BadRequest, Description = "Request is malformed.", ShowSchema = false)]
+        [Response(HttpStatusCode = (int)HttpStatusCode.Unauthorized, Description = "API key is unknown or invalid.", ShowSchema = false)]
+        [Response(HttpStatusCode = (int)HttpStatusCode.Forbidden, Description = "Insufficient access to this Employment progression.", ShowSchema = false)]
+        [Response(HttpStatusCode = (int)422, Description = "Employment progression validation error(s).", ShowSchema = false)]
+        [ProducesResponseType(typeof(Models.EmploymentProgression), (int)HttpStatusCode.OK)]
+        [Display(Name = "Get", Description = "Ability to retrieve an individual employment progression for the given customer.")]
+        public async Task<HttpResponseMessage> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = RouteValue)]
+            HttpRequest req, ILogger logger, string customerId, string EmploymentProgressionId)
+        {
+            _loggerHelper.LogMethodEnter(logger);
+
+            var correlationId = _httpRequestHelper.GetDssCorrelationId(req);
+
+            var guidHelper = new GuidHelper();
+            var correlationGuid = guidHelper.ValidateGuid(correlationId);
+
+            var touchpointId = _httpRequestHelper.GetDssTouchpointId(req);
+            if (string.IsNullOrEmpty(touchpointId))
+            {
+                _loggerHelper.LogInformationMessage(logger, correlationGuid, "Unable to locate 'TouchpointId' in request header.");
+                return _httpResponseMessageHelper.BadRequest();
+            }
+
+            var ApimURL = _httpRequestHelper.GetDssApimUrl(req);
+            if (string.IsNullOrEmpty(ApimURL))
+            {
+                _loggerHelper.LogInformationMessage(logger, correlationGuid, "Unable to locate 'apimurl' in request header");
+                return _httpResponseMessageHelper.BadRequest();
+            }
+
+            if (!Guid.TryParse(customerId, out var customerGuid))
+            {
+                _loggerHelper.LogInformationMessage(logger, correlationGuid, $"Unable to parse 'customerId' to a Guid: {customerId}");
```

Comment:
I have another way to do this check sdd-13 on diversity v3 branch. look at the GuidHelper Class in Common